app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int, *args) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people_list: list) -> list:
+
+    persons = [Person(per["name"], per["age"]) for per in people_list]
+
+    for people, value in zip(persons, people_list):
+        if "wife" in value and value["wife"]:
+            for count in Person.people:
```

Comment:
why do you need this loop? you can get the necessary instance from the dictionary by its key Person.people[value["wife"]]

Reply:
Corrected on all counts :)